test/integration/stats_integration_test.cc

```diff
@@ -273,7 +273,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // 2020/05/13  10531    44425       44600   Refactor resource manager
   // 2020/05/20  11223    44491       44600   Add primary clusters tracking to cluster manager.
   // 2020/06/10  11561    44491       44811   Make upstreams pluggable
-  // 2020/04/23  10661    44425       46000   per-listener connection limits
```

Comment:
latest CI failure was this test again (second time this PR has failed CI only due to stats numbers)
I'm really not sure what to do about the per-listener limit, which had a size and date from the setec upstream PR which were both out of kilter with actual dates and sizes so I just removed it.

I still question the value of this test being worth the toil, especially if it's going to be confusing and inaccurate for things like backports :-/